Basic support for histograms (using matplotlib `hist`)

ruivieira opened the pull request:
TODO: Support for additional `hist()` arguments.

stisa commented:
Hi, thanks for the PR!
Since you import `strutils`, what do you think about reducing the whole proc to
something like this?
```nim
discard runSimpleString(
  "pp.hist($#, bins=$#, log=$#)" % [
    $x,
    if bins != 0: $bins else: "None", 
    capitalizeAscii($log)
  ]
)
```
I guess `bins` can't be negative, so `Natural` may be better for its type?

Also, do you want me to wait for you to add support for the other arguments before merging?

Thanks again!

ruivieira commented:
@stisa np!

It looks better indeed. I've changed the `hist` code. I've also added an example to the notebook if you don't mind.

The histogram support is basic, but it works, so I'm for merging now in case others want to use it.
I can add the other arguments later on.

Thanks!
